en/docs/deploy/api-microgateway/service-discovery.md

```diff
@@ -1,140 +1,91 @@
-    ```
-    
-    ``` bash tab="Example - docker"
-        docker run -d -p 9090:9090 -p 9095:9095 -e etcdurl=http://127.0.0.1:2379 -e etcdtimer=10000 docker.wso2.com/petstore:v1
-    ```
-    
-    - `<etcd-url>` - Provide the etcd URL. If the etcd URL is secured (https), you need to add the etcd server certificate to the [Ballerina](https://ballerina.io/) truststore.
-
-    - `<etcd-timer>` - This is the time period for the periodic timer task (in milliseconds). The default time period is 10 seconds. This parameter specifies the time interval for each periodic query to etcd; therefore, this value should be lower than the TTL of the token. etcd supports the following types of tokens.
-
-        - **Simple tokens**  
-            These tokens have a default time-to-live (TTL) of 5 minutes, but the TTL is configurable. In addition, the TTL refreshes each time a request goes to etcd.
-
-            Example:
-        If etcd is configured to use simple tokens with a TTL of 5 minutes, the `etcdTimer` parameter value should contain a value less than 5 minutes. If the etcdTimer value is 24000 (24000 ms = 4 minutes). Every 4 minutes the timer task is triggered, the etcd is queried, and it refreshes the TTL of the token.
-
-        -   **JWT tokens**  
-        This token does not have a TTL.
-    
-
-### Changing the API endpoint URL via etcd
-
-For example, If the production endpoint URL changes and the corresponding etcd key is `etcddemoprod` , update the value related to the `etcddemoprod` key on the etcd server.
-If the periodic timer task value is 10 seconds, within 10 seconds the periodic task gets executed and updates WSO2 API Microgateway with the new value. Thereby, in this scenario, if you are in the process of querying an API, WSO2 API Microgateway redirects your request to the new URL of the endpoint within 10 seconds.
-
-Follow the instructions below to update the etcd with your new API endpoint URL:
-
-   1.  Put the new endpoint URL for the corresponding key in the etcd server using the following command.
-   
-       ``` java tab="Format"
-       etcdctl put <Key> <Value>
-       ```
-       
-       ``` java tab="Example"
-       etcdctl put petstore http://petstore.swagger.io/v2/
-       ```
+        1. The adapter takes one `pollInterval` amount of time to update the upstreams' data to the Router.
+        During that time requests that arrive at the Microgateway are served from the
+        `default_host`. <br>
+        2. Consul upstreams can be set both on `x-wso2-production-endpoints` and `x-wso2-sandbox-endpoints`.<br>
+        3. Microgateway supports Both API level and Resource level endpoints for Consul service discovery.<br>
+        4. Type under the vendor extension should be `loadbalance`.
+        <br>
+        5. Upon successfully [deploying your API](quick-start-guide.md#step-2---create-and-deploy-an-api-project), the Adapter will poll the Consul HTTP API for changes concerning the services.
```

Comment:
path is wrong. It has to be as follows:
{{base_path}}/get-started/quick-start-guide/quick-start-guide/#step-2-publish-the-api